logrotate/src/LogRotateHelper.ipp

```diff
@@ -30,6 +30,32 @@
 #include <sstream>
 
 
+#if (defined(WIN32) || defined(_WIN32) || defined(__WIN32__)) && !defined(__MINGW32__)
+
+//http://stackoverflow.com/questions/321849/strptime-equivalent-on-windows
+
+#include <time.h>
+#include <iomanip>
+
+char* strptime(const char* s,
```

Comment:
A similar function also exists in g3log/src/time.cpp. It's a re-implementation of ```std::put_time```
https://github.com/KjellKod/g3log/blob/master/src/time.cpp